lib/msf/core/post/linux/system.rb

```diff
@@ -197,6 +216,17 @@ def pidof(program)
     pids
   end
 
+  #
+  # Gets the uid of a pid
+  # @return [String]
+  #
+  def pid_uid(pid)
```

Comment:
May as well use string interpolations, as you're using double quotes, calling `to_s` on `pid` then using string concatenation.

Also, the `return` is redundant.

Refactored:

```ruby
  def pid_uid(pid)
    read_file("/proc/#{pid}/status").to_s
  end
```